benchmarks/transport/CMakeLists.txt

```diff
@@ -11,23 +11,11 @@ if (GHEX_USE_UCP)
         add_executable(${_t} ${_t}_mt.cpp )
         target_compile_definitions(${_t} PRIVATE USE_HEAVY_CALLBACKS USE_RAW_SHARED_MESSAGE USE_POOL_ALLOCATOR)
         target_link_libraries(${_t} ghexlib)
-        if (GHEX_USE_PMIX)
```

Comment:
I see, so the main benefit would be to avoid this repetition when setting up the targets in Cmake, right?

Reply:
@bettiolm precisely. all config is is one place, so once you add a cmake variable, you will not need to go to all subdirectories and update the targets.